source/administration/config-settings.rst

```diff
@@ -1107,10 +1109,10 @@ Enable Encryption
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 **True**: Mattermost will decrypt SAML Assertions encrypted with your Service Provider Public Certificate.
 
-**False**: Encyption is disabled.
+**False**: Encyption is disabled. Not recommended for production environments. For testing only.
```

Comment:
typo... should be "Encryption". But "Encryption is disabled." doesn't seem like the correct statement anyway. When `true` decryption occurs, so when `false`, then decryption is not occurring. Maybe "Decryption is disabled." is correct?

Removing that sentence might be more emphatic.